### DISCUSSION TITLE
No separation between dependencies and devDependencies?

### DISCUSSION THREAD

**Discussion by bfelbo** [statelyai/xstate#3395] [General] [answered]
I'm curious if there's a reason why xstate doesn't use the `devDependencies` key for e.g. ts-jest in the [package.json](https://github.com/statelyai/xstate/blob/main/package.json#L32)?

https://stackoverflow.com/a/22004559/1452257 describes some of the benefits of separating production dependencies vs. dev dependencies.

**Comment by davidkpiano** [comment, marked as answer]
You are looking at the dependencies for `xstate-monorepo`; this is not published.

The published `xstate` library itself has _zero_ dependencies.

**Reply by bfelbo** [reply]
I see, thanks. Appreciate the quick response!

For others seeing this, here's the [package.json](https://github.com/statelyai/xstate/blob/main/packages/core/package.json) for the `xstate` library itself.
